.terraform.lock.hcl

```diff
@@ -0,0 +1,152 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
https://www.terraform.io/docs/language/dependency-lock.html#lock-file-location -> Should be part of version control